gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/FeatureDetailsChromeBehavior.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.ui.home.featuredetails;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.coordinatorlayout.widget.CoordinatorLayout;
+import com.google.android.gnd.R;
+import com.google.android.gnd.ui.home.BottomSheetDependentBehavior;
+import com.google.android.gnd.ui.home.BottomSheetMetrics;
+
+public class FeatureDetailsChromeBehavior extends BottomSheetDependentBehavior<ViewGroup> {
+  public FeatureDetailsChromeBehavior(Context context, AttributeSet attrs) {
+    super(context, attrs);
+  }
+
+  @Override
+  protected void onBottomSheetChanged(
```

Comment:
Consider adding comment here to make it clear when this method will be called.

Reply:
Added more JavaDoc to the base class. Is that sufficient / better than duplicating JavaDoc in multiple places? Note that the IDE is able to help here.